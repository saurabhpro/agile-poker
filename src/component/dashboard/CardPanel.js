import React from 'react';
import AllCardsPanel from '../poker-card/AllCardsPanel';

export default function CardPanel() {
  return (
    <div>
      <AllCardsPanel />
    </div>
  );
}
