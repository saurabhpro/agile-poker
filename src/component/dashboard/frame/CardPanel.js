import React from 'react';
import AllCardsPanel from './poker-card/AllCardsPanel';

export default function CardPanel({currentUser}) {
  return (
    <div>
      <AllCardsPanel currentUser={currentUser}/>
    </div>
  );
}
