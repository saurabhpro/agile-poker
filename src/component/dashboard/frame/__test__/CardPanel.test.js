import React from 'react';
import { shallow } from 'enzyme';
import CardPanel from '../CardPanel';

let component;

describe('center card panel', () => {
  beforeAll(() => {
    component = shallow(<CardPanel />);
    //console.debug("This was prited",component.debug());
  });

  it('matches the rendered snapshot', () => {
    expect(component).toMatchSnapshot();
  });


  it('should render one <AllCardsPanel>', () => {
    expect(component.find('AllCardsPanel')).toHaveLength(1);
  });
});
