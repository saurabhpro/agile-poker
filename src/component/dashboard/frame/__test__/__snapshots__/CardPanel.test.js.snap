// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`center card panel matches the rendered snapshot 1`] = `
<div>
  <AllCardsPanel />
</div>
`;
